mavros_extras/src/plugins/distance_sensor.cpp

```diff
@@ -330,6 +332,9 @@ DistanceSensorItem::Ptr DistanceSensorItem::create_item(DistanceSensorPlugin *ow
 		// lookup for numeric value
 		p->orientation = utils::sensor_orientation_from_str(orientation_str);
 
+	// Variance window
+	pnh.param("variance_window_size", p->variance_window_size, DEFAULT_WINDOW_SIZE);
```

Comment:
Note: because of `size_t` change, you should provide type to param template, something like `pnh.param<unsigned int>(...)`.